scripts/paypal/payment-reconciliator.ts

```diff
@@ -460,7 +461,7 @@ const findOrphanSubscriptions = async (_, commander) => {
       }
 
       // Make sure all transactions exist in the ledger
-      for (const paypalTransaction of <Record<string, unknown>[]>response['transactions']) {
+      for (const paypalTransaction of <PaypalTransaction[]>response['transactions']) {
```

Comment:
I would rather type `fetchPaypalTransactionsForSubscription` result.
To make sure it is backward compatible and easy, you can type that function by combining a generic with a property type:
```ts
const fetchPaypalTransactionsForSubscription = async (host, subscriptionId): Promise<Record<string, unknown> & { transactions?: PaypalTransaction[] }> =>
```

Reply:
Good call